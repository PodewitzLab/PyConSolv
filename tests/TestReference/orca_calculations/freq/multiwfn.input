./orca_freq.molden.input
7
18
1




y
0
0
q
