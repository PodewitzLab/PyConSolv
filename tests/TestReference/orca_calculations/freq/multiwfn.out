 Error: Unable to find the input file you specified in argument
 
 Multiwfn -- A Multifunctional Wavefunction Analyzer
 Version 3.8(dev), release date: 2022-Dec-1
 Developer: Tian Lu (Beijing Kein Research Center for Natural Sciences)
 Below paper ***MUST BE CITED*** if Multiwfn is utilized in your work:
          Tian Lu, Feiwu Chen, J. Comput. Chem., 33, 580-592 (2012)
 See "How to cite Multiwfn.pdf" in Multiwfn binary package for more information
 Multiwfn official website: http://sobereva.com/multiwfn
 Multiwfn English forum: http://sobereva.com/wfnbbs
 Multiwfn Chinese forum: http://bbs.keinsci.com/wfn

 ( Number of parallel threads:   8  Current date: 2023-09-01  Time: 17:24:05 )
 
 Input file path, for example E:\Gun_Gale_Online\Kirito.wfn
 (Supported: .mwfn/wfn/wfx/fch/molden/31/chg/pdb/xyz/mol/mol2/cif/cub, etc.)
 Hint: Press ENTER button directly can select file in a GUI window. To reload the file last time used, simply input the letter "o". Input such as ?miku.fch can open the miku.fch in the same folder as the file last time used.
 Please wait...
 Loading various information of the wavefunction
 This file is recognized to be generated by ORCA because there is "orca" word in title line. Special treatments are applied...
 Loading basis set definition...
 All D basis functions are spherical harmonic type
 All F basis functions are spherical harmonic type
 Loading orbitals...
 The actual number of orbitals read:       126
 Converting basis function information to GTF information...
 Back converting basis function information from Cartesian to spherical type...
 Generating density matrix...
 Generating overlap matrix...
 
 Total/Alpha/Beta electrons:     72.0000     36.0000     36.0000
 Net charge:     0.00000      Expected multiplicity:    1
 Atoms:     11,  Basis functions:    126,  GTFs:    231
 This is a restricted single-determinant wavefunction
 Orbitals from 1 to    36 are occupied
 Loading electron density functions (EDF) information from built-in EDF library
 (The library is available at https://github.com/zorkzou/Molden2AIM)
 Pt(    1)      Core electrons: 60     EDF primitive GTFs: 36
 The number of total inner-core electrons:    60
 The number of total EDF primitive GTFs:    36
 Loading EDF library finished!

 Loaded ./orca_freq.molden.input successfully!
 
 Formula: H6 N2 Cl2 Pt1 
 Molecule weight:       300.05138 Da
 Point group: C1 
 
 "q": Exit program gracefully          "r": Load a new file
                    ************ Main function menu ************
 0 Show molecular structure and view orbitals
 1 Output all properties at a point       2 Topology analysis
 3 Output and plot specific property in a line
 4 Output and plot specific property in a plane
 5 Output and plot specific property within a spatial region (calc. grid data)
 6 Check & modify wavefunction
 7 Population analysis and calculation of atomic charges
 8 Orbital composition analysis           9 Bond order analysis
 10 Plot total DOS, partial DOS, OPDOS, local DOS and photoelectron spectrum
 11 Plot IR/Raman/UV-Vis/ECD/VCD/ROA/NMR spectrum
 12 Quantitative analysis of molecular surface
 13 Process grid data (No grid data is presented currently)
 14 Adaptive natural density partitioning (AdNDP) analysis
 15 Fuzzy atomic space analysis
 16 Charge decomposition analysis (CDA) and plot orbital interaction diagram
 17 Basin analysis                       18 Electron excitation analysis
 19 Orbital localization analysis        20 Visual study of weak interaction
 21 Energy decomposition analysis        22 Conceptual DFT (CDFT) analysis
 23 ETS-NOCV analysis                    24 (Hyper)polarizability analysis
 25 Electron delocalization and aromaticity analyses
 100 Other functions (Part 1)            200 Other functions (Part 2)
 300 Other functions (Part 3)
 
      ============== Population analysis and atomic charges ==============
 -2 Calculate interaction energy between fragments based on atomic charges
 -1 Define fragment
 0 Return
 1 Hirshfeld atomic charge
 2 Voronoi deformation density (VDD) atom population
 5 Mulliken atom & basis function population analysis
 6 Lowdin atom & basis function population analysis
 7 Modified Mulliken atom population defined by Ros & Schuit (SCPA)
 8 Modified Mulliken atom population defined by Stout & Politzer
 9 Modified Mulliken atom population defined by Bickelhaupt
 10 Becke atomic charge with atomic dipole moment correction
 11 Atomic dipole corrected Hirshfeld atomic charge (ADCH) (recommended)
 12 CHELPG ESP fitting atomic charge
 13 Merz-Kollmann (MK) ESP fitting atomic charge
 14 AIM atomic charge
 15 Hirshfeld-I atomic charge
 16 CM5 atomic charge    -16 Generate 1.2*CM5 atomic charge
 17 Electronegativity Equalization Method (EEM) atomic charge
 18 Restrained ElectroStatic Potential (RESP) atomic charge
 19 Gasteiger (PEOE) charge
 
             ------------ Calculation of RESP charges ------------
 -1 Load list of conformer and weights from external file
 0 Return
 1 Start standard two-stage RESP fitting calculation
 2 Start one-stage ESP fitting calculation with constraints
 3 Set method and parameters for distributing fitting points, current: MK
 4 Set hyperbolic penalty and various other running parameters
 5 Set equivalence constraint in fitting, current: H in CH2 and CH3
 6 Set charge constraint in fitting, current: No constraint
 7 Set the way of determining connectivity, current: Guess from bond length
 8 Toggle if loading fitting points and ESP values from Gaussian output file of pop=MK/CHELPG task with IOp(6/33=2) during the calculation, current: No
 9 Load additional fitting centers, current: None
 10 Choose the atomic radii used in fitting, current: Automatic
 11 Choose ESP type, current: Nuclear + Electronic
 Atomic radii used:
 Element:H      vdW radius (Angstrom): 1.200
 Element:N      vdW radius (Angstrom): 1.500
 Element:Cl     vdW radius (Angstrom): 1.700

 vdW radius used in fitting for Pt is missing, input it in Angstrom, e.g. 1.7
 Hint: If pressing ENTER button directly, corresponding UFF radii scaled by 1/1.2 will be used, which usually is a proper workaround. (If you want to automatically employ this treatment, setting "ispecial" in settings.ini to 1)
 Radius of Pt has been set to  1.1475 Angstrom
 Number of MK fitting points used:      8968
 Calculating ESP at fitting points, please wait...
 
 Initializing LIBRETA library (fast version) for ESP evaluation ...
 LIBRETA library has been successfully initialized!

 NOTE: The ESP evaluation code based on LIBRETA library is being used. Please *NOT ONLY* cite Multiwfn original paper (J. Comput. Chem., 33, 580-592 (2012)) in your work, *BUT ALSO* cite the paper describing the efficient ESP evaluation algorithm adopted by Multiwfn:
 Jun Zhang, Tian Lu, Phys. Chem. Chem. Phys., 23, 20323 (2021) DOI: 10.1039/D1CP02805G
 
 Progress: [--------------------------------------------------]    0.00 %     - Progress: [#-------------------------------------------------]    3.35 %     \ Progress: [#-------------------------------------------------]    3.36 %     | Progress: [#####---------------------------------------------]   10.04 %     / Progress: [#####---------------------------------------------]   10.05 %     - Progress: [########------------------------------------------]   16.73 %     \ Progress: [########------------------------------------------]   16.74 %     | Progress: [###########---------------------------------------]   23.42 %     / Progress: [###########---------------------------------------]   23.43 %     - Progress: [###############-----------------------------------]   30.11 %     \ Progress: [################----------------------------------]   33.45 %     | Progress: [##################--------------------------------]   36.80 %     / Progress: [##################--------------------------------]   36.81 %     - Progress: [#####################-----------------------------]   43.49 %     | Progress: [#####################-----------------------------]   43.50 %     \ Progress: [#########################-------------------------]   50.18 %     / Progress: [##########################------------------------]   53.52 %     - Progress: [##########################------------------------]   53.53 %     \ Progress: [##############################--------------------]   60.21 %     | Progress: [##############################--------------------]   60.23 %     / Progress: [##############################--------------------]   60.24 %     - Progress: [###################################---------------]   70.25 %     \ Progress: [###################################---------------]   70.26 %     | Progress: [######################################------------]   76.94 %     / Progress: [########################################----------]   80.29 %     - Progress: [########################################----------]   80.30 %     \ Progress: [###########################################-------]   86.98 %     | Progress: [#############################################-----]   90.32 %     / Progress: [##############################################----]   93.67 %     - Progress: [##############################################----]   93.68 %     \ Progress: [##############################################----]   93.69 %     | Progress: [##################################################]  100.00 %     / 
 Calculation of ESP took up wall clock time         0 s
 
 No charge constraint is imposed in this stage
 No atom equivalence constraint is imposed in this fitting stage
 
 **** Stage 1: RESP fitting under weak hyperbolic penalty
 Convergence criterion:  0.0000010000
 Hyperbolic restraint strength (a): 0.000500    Tightness (b): 0.100000
 Iter:   1   Maximum charge variation:    0.5138731006
 Iter:   2   Maximum charge variation:    0.0302588446
 Iter:   3   Maximum charge variation:    0.0004381733
 Iter:   4   Maximum charge variation:    0.0000061081
 Iter:   5   Maximum charge variation:    0.0000000849
 Successfully converged!
 
 **** Stage 2: RESP fitting under strong hyperbolic penalty
 Stage 2 of standard RESP fitting is skipped since no atom needs to be fitted
 
   Center       Charge
     1(Pt)  -0.0497105584
     2(N )  -0.5445760109
     3(H )   0.3447584260
     4(H )   0.3441152090
     5(H )   0.3117300280
     6(N )  -0.5341066426
     7(H )   0.3410737786
     8(H )   0.3087176992
     9(H )   0.3422905444
    10(Cl)  -0.4322028118
    11(Cl)  -0.4320896615
 Sum of charges:   0.0000000000
 RMSE:    0.003819   RRMSE:    0.067447
 
 If outputting atom coordinates with charges to orca_freq.molden.chg in current folder? (y/n)
 Result have been saved to orca_freq.molden.chg in current folder
 Columns 1 to 5 are name,X,Y,Z,charge respectively, coordinates are in Angstrom
 
             ------------ Calculation of RESP charges ------------
 -1 Load list of conformer and weights from external file
 0 Return
 1 Start standard two-stage RESP fitting calculation
 2 Start one-stage ESP fitting calculation with constraints
 3 Set method and parameters for distributing fitting points, current: MK
 4 Set hyperbolic penalty and various other running parameters
 5 Set equivalence constraint in fitting, current: H in CH2 and CH3
 6 Set charge constraint in fitting, current: No constraint
 7 Set the way of determining connectivity, current: Guess from bond length
 8 Toggle if loading fitting points and ESP values from Gaussian output file of pop=MK/CHELPG task with IOp(6/33=2) during the calculation, current: No
 9 Load additional fitting centers, current: None
 10 Choose the atomic radii used in fitting, current: Automatic
 11 Choose ESP type, current: Nuclear + Electronic
 
      ============== Population analysis and atomic charges ==============
 -2 Calculate interaction energy between fragments based on atomic charges
 -1 Define fragment
 0 Return
 1 Hirshfeld atomic charge
 2 Voronoi deformation density (VDD) atom population
 5 Mulliken atom & basis function population analysis
 6 Lowdin atom & basis function population analysis
 7 Modified Mulliken atom population defined by Ros & Schuit (SCPA)
 8 Modified Mulliken atom population defined by Stout & Politzer
 9 Modified Mulliken atom population defined by Bickelhaupt
 10 Becke atomic charge with atomic dipole moment correction
 11 Atomic dipole corrected Hirshfeld atomic charge (ADCH) (recommended)
 12 CHELPG ESP fitting atomic charge
 13 Merz-Kollmann (MK) ESP fitting atomic charge
 14 AIM atomic charge
 15 Hirshfeld-I atomic charge
 16 CM5 atomic charge    -16 Generate 1.2*CM5 atomic charge
 17 Electronegativity Equalization Method (EEM) atomic charge
 18 Restrained ElectroStatic Potential (RESP) atomic charge
 19 Gasteiger (PEOE) charge
 
 "q": Exit program gracefully          "r": Load a new file
                    ************ Main function menu ************
 0 Show molecular structure and view orbitals
 1 Output all properties at a point       2 Topology analysis
 3 Output and plot specific property in a line
 4 Output and plot specific property in a plane
 5 Output and plot specific property within a spatial region (calc. grid data)
 6 Check & modify wavefunction
 7 Population analysis and calculation of atomic charges
 8 Orbital composition analysis           9 Bond order analysis
 10 Plot total DOS, partial DOS, OPDOS, local DOS and photoelectron spectrum
 11 Plot IR/Raman/UV-Vis/ECD/VCD/ROA/NMR spectrum
 12 Quantitative analysis of molecular surface
 13 Process grid data (No grid data is presented currently)
 14 Adaptive natural density partitioning (AdNDP) analysis
 15 Fuzzy atomic space analysis
 16 Charge decomposition analysis (CDA) and plot orbital interaction diagram
 17 Basin analysis                       18 Electron excitation analysis
 19 Orbital localization analysis        20 Visual study of weak interaction
 21 Energy decomposition analysis        22 Conceptual DFT (CDFT) analysis
 23 ETS-NOCV analysis                    24 (Hyper)polarizability analysis
 25 Electron delocalization and aromaticity analyses
 100 Other functions (Part 1)            200 Other functions (Part 2)
 300 Other functions (Part 3)
